mmengine/optim/optimizer/optimizer_wrapper.py

```diff
@@ -161,6 +162,14 @@ def __init__(self,
         # the loss factor will always be the same as `_accumulative_counts`.
         self._remainder_counts = -1
 
+        # add a tensor to the optimizer to retrieve global lr
+        if hasattr(self.optimizer, 'defaults'):
+            new_param_settings = {
```

Comment:
I suggest saving the new_param_settings as an attribute of OptimWrapper, such as `self.tracker_group = {...}`. This would simplify the implementation of load_state_dict, get_lr, and get_momentum.